controller/cache/cache.go

```diff
@@ -69,6 +70,11 @@ const (
 
 	// EnvClusterCacheRetryUseBackoff is the env variable to control whether to use a backoff strategy with the retry during cluster cache sync
 	EnvClusterCacheRetryUseBackoff = "ARGOCD_CLUSTER_CACHE_RETRY_USE_BACKOFF"
+
+	// AnnotationApplyResourcesUpdate when set to true on a resource that is not tracked under an app, argocd will generate a
+	// hash and apply `ignoreResourceUpdate` configuration on it. If the annotation is set to false (or not presented) on a resource
+	// that is not tracked under an app, ignoreResourceUpdates configuration will not be applied.
+	AnnotationApplyResourcesUpdate = "argocd.argoproj.io/apply-resources-update"
```

Comment:
Now that the code is written, I think it will be much easier for users to have the same terminology with the existing feature. wdyt?

```suggestion
	AnnotationIgnoreResourceUpdates = "argocd.argoproj.io/ignore-resource-updates"
```

Reply:
@agaudreault I think `AnnotationApplyResourcesUpdate` is more logical to me. `AnnotationIgnoreResourceUpdates` seems the invert of what we are doing and therefore we need to flip our logic if we rename it.